npm-shrinkwrap.json

```diff
@@ -2649,11 +2624,11 @@
       }
     },
     "@wordpress/autop": {
-      "version": "2.0.2",
-      "resolved": "https://registry.npmjs.org/@wordpress/autop/-/autop-2.0.2.tgz",
-      "integrity": "sha512-YoaLK8TJVvD6e0t5VbiwPvkuIyBim5WyDXgE6Kg6T9xLWDEu+Azp1ljk/e+D6rPJoC895eBQZjCWxrch5WFvLw==",
+      "version": "2.1.0",
```

Comment:
these upgrades are all minorly scary. Probably need to check gutenlypso quick?

Reply:
I tried the block editor within Calypso quickly and it seemed okay (no garbled paragraphs) -- but it's no longer Gutenlypso anymore anyway, is it? (see p7DVsv-69v-p2)

Reply:
Accessible via feature flag: `?flags=-calypsoify/iframe`

Reply:
I've just done some smoke testing and it seems to work fine 👍